setup.py

```diff
@@ -3,8 +3,8 @@
 from setuptools import setup, find_packages
 
 setup(
-    name="python-magnetpanel",
-    version="0.7.3",
+    name="magnetpanel",
+    version="0.7.4",
```

Comment:
You should also add `install_requires=["taurus"]` in the setup.py (was in setup.cfg before).

Reply:
Thanks, added